.github/workflows/pipeline.yml

```diff
@@ -24,15 +24,21 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: "3.6"
-      - name: Build Windows binary
-        run: dotnet publish -c Release -r win-x64 -o Build/Release/ /nowarn:cs0618
+      - name: Build Windows .net 5 binary
+        run: dotnet publish -c Release -f net5.0-windows -r win-x64 -o Build/Release/net5/ /nowarn:cs0618
+      - name: Build Windows net452 binary
+        run: dotnet publish libMBIN-DLL -c Release -f net452 -r win-x64 -o Build/Release/net4/ /nowarn:cs0618
+      - name: Move some files to rename them
+        run: |
+          cp Build/Release/net5/MBINCompiler.exe Build/Release/MBINCompiler.exe
+          cp Build/Release/net4/libMBIN.dll Build/Release/net4/libMBIN-dotnet4.dll
```

Comment:
Any particular reason to rename it libMBIN-dotnet4.dll?  The fact that it is in Release/net4 folder should be enough to identify it apart from the net5 version.

Reply:
when it gets released on github it will just get the filename, not the path (like, when you see the attached resources in a release it has just file names). So even though it doesn't matter now, at some point I need to rename it so that there aren't two `libMBIN.dll` files in the list of release files, and since I have to move the built file anyway I figured I'd do it now...

Reply:
Ok, it is needed for the release.  No problem then, we'll re-rename it after the download from releases!